### ISSUE TITLE
[UIComponents] Hardware back button issue

### ISSUE THREAD

**Issue by DariuszPaziewski** [Samsung/TAU#663]
The application performs two steps back in history when you are deeper than the first level in the navigation tree and the hardware back button is used. It occurs on wearable 5.5 emulator and app with TAU 1.1.15. It does not occur on wearable 5.0.

Emulator version:
Model=Tizen5/Unified;
Build=tizen-unified_20191031.1;
Release=20191031.1;
[Build]
Type=eng;
Date=20191031_052051;
Time=05:20:51;
Variant=NONE;
ID=tizen-unified;

Steps to reproduce:
1. Make sure you use TAU 1.1.15
2. Launch the UIComponents app
3. Go to Navigation > IndexScrollBar
4. The IndexScrollbar page is displayed
5. Press the hardware back button once
6. You are in the main menu (two steps up)

Expected result:
After pressing the hardware back button the app should navigate only one step up to the Navigation page.

**Comment by lslachciak** [comment]
confirmed, reproducible on 5.5 emulator, not reproducible on 5.0 emulator

**Comment by lslachciak** [comment]
I prepared sample Tizen app with just content and internal links.
[UIComponents.zip](https://github.com/Samsung/TAU/files/4231404/UIComponents.zip)
It occurred that on 5.5 emulator window.history.back is called automatically when HW back key is pressed.
If I run this simple app on 5.0 no navigation back occurs after pressing window.historyback

Our sample UIComponents app for 1.1 has event listener for tizen_hwback key which calls window.history.back. That is why double back was done. Probably we need to remove this aditional listener for 5.5 from sample UIComponents app

**Comment by lslachciak** [comment]
Similar situation is happening with mobile UIComponents app and mobile 5.5 simulator
[Version]
Model=Tizen5/Unified;
Build=tizen-unified_20191031.1;
Release=20191031.1;

I know how to fix this issue -
in UIComponents app , file app.js we shouldnt call window.history.back.
```
diff --git a/examples/mobile/UIComponents/js/app.js b/examples/mobile/UIComponents/js/app.js
index f27f2df..e98fe2c 100644
--- a/examples/mobile/UIComponents/js/app.js
+++ b/examples/mobile/UIComponents/js/app.js
@@ -14,8 +14,6 @@
                                        tizen.application.getCurrentApplication().exit();
                                } catch (ignore) {
                                }
-                       } else {
-                               window.history.back();
                        }
                }
        });
diff --git a/examples/wearable/UIComponents/js/app.js b/examples/wearable/UIComponents/js/app.js
index f27f2df..e98fe2c 100644
--- a/examples/wearable/UIComponents/js/app.js
+++ b/examples/wearable/UIComponents/js/app.js
@@ -14,8 +14,6 @@
                                        tizen.application.getCurrentApplication().exit();
                                } catch (ignore) {
                                }
-                       } else {
-                               window.history.back();
                        }
                }
        });

```
But before fix is published I need to consider how this influence code.tizen.org

**Comment by lslachciak** [comment]
The good news is that on code.tizen.org DE adds back-buttonsupport.js file so it is handling back command in its own way.

**Comment by lslachciak** [comment]
As mentioned in PR https://github.com/Samsung/TAU/pull/665
we need to clarify this issue with WRT

**Comment by lslachciak** [comment]
This was WRT issue and was fixed in Tizen 5.0 wearable emulator. Please check latest 5.0 wearable emulator for Tizen (not 5.5)  Issue is not TAU issue.

**Comment by lslachciak** [comment]
emulator 5.0 which is working:
[Version]
Model=Tizen5/Unified;
Build=tizen-5.0-unified_20181207.1;
Release=20181207.1;
[Build]
Type=eng;
Date=20181207_110549;
Time=11:05:49;
Variant=NONE;
ID=tizen-5.0-unified;
sh-3.2$
